[2014-03-10 09:28:58] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42703]: Undefined column: 7 ERROR:  column ma_map.merchatagreement_id does not exist LINE 1: ...oin "map" on "map"."id" = "ma_map"."map_id" where "ma_map"."...                                                              ^' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:274 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(274): PDOStatement->execute(Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\PostgresConnection), 'select * from "...', Array) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select * from "...', Array, Object(Closure)) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select * from "...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /var/www/arp_dev/app/models/Merchantagreement.php(34): Illuminate\Database\Query\Builder->get() #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(189): Merchantagreement->paramval('percentage') #8 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #9 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #13 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #20 {main} Next exception 'Exception' with message 'SQLSTATE[42703]: Undefined column: 7 ERROR:  column ma_map.merchatagreement_id does not exist LINE 1: ...oin "map" on "map"."id" = "ma_map"."map_id" where "ma_map"."...                                                              ^ (SQL: select * from "ma_map" inner join "map" on "map"."id" = "ma_map"."map_id" where "ma_map"."merchatagreement_id" = ? and "map"."parameter" = ?) (Bindings: array (   0 => 6,   1 => 'percentage', ))' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select * from "...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select * from "...', Array, Object(Closure)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select * from "...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /var/www/arp_dev/app/models/Merchantagreement.php(34): Illuminate\Database\Query\Builder->get() #6 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(189): Merchantagreement->paramval('percentage') #7 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #8 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #12 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #18 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #19 {main} [] []
[2014-03-10 09:29:22] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:189 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(189): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 189, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:29:40] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:190 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(190): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 190, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:39:37] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:192 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(192): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 192, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:42:09] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:192 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(192): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 192, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:43:18] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:193 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(193): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 193, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:43:57] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ','' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:191 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 09:44:05] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:191 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(191): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 191, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:44:52] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:191 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(191): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 191, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:45:10] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:194 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(194): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 194, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:46:26] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:192 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(192): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 192, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:47:16] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:192 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(192): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 192, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:53:21] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Database\Eloquent\Collection::where()' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:189 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 09:54:34] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Collection::$percentage' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:189 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(189): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 189, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:54:47] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Illuminate\Database\Eloquent\Collection::select()' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:189 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 09:57:03] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Collection::$map_id' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:189 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(189): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 189, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 09:57:41] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Collection::$map_value' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:189 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(189): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 189, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 10:04:20] log.ERROR: exception 'ErrorException' with message 'Undefined property: Illuminate\Database\Eloquent\Collection::$map_value' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:187 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(187): Illuminate\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/arp_de...', 187, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 10:10:07] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '->' (T_OBJECT_OPERATOR)' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:187 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 10:11:45] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:187 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(187): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 187, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 10:12:10] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:187 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(187): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 187, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 10:12:35] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:187 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(187): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/arp_de...', 187, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #2 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #6 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #13 {main} [] []
[2014-03-10 10:27:08] log.ERROR: exception 'ErrorException' with message 'Undefined variable: specificationrows' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:230 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(230): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 230, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1000') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 10:27:42] log.ERROR: exception 'ErrorException' with message 'Undefined variable: specificationrows' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:230 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(230): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 230, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1000') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 10:27:44] log.ERROR: exception 'ErrorException' with message 'Undefined variable: specificationrows' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:230 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(230): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 230, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1000') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 10:30:00] log.ERROR: exception 'ErrorException' with message 'Undefined variable: costrows' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:266 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(266): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 266, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1000') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 11:52:10] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42703]: Undefined column: 7 ERROR:  column "merchantagreement_id" does not exist LINE 1: select * from "mainvoice" where "merchantagreement_id" = $1 ...                                         ^' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:274 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(274): PDOStatement->execute(Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\PostgresConnection), 'select * from "...', Array) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select * from "...', Array, Object(Closure)) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select * from "...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(441): Illuminate\Database\Query\Builder->get(Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(132): Illuminate\Database\Eloquent\Builder->getModels(Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(108): Illuminate\Database\Eloquent\Builder->get(Array) #9 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(677): Illuminate\Database\Eloquent\Builder->first() #10 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getLastbalance(6) #11 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #12 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #16 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #20 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #21 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #22 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #23 {main} Next exception 'Exception' with message 'SQLSTATE[42703]: Undefined column: 7 ERROR:  column "merchantagreement_id" does not exist LINE 1: select * from "mainvoice" where "merchantagreement_id" = $1 ...                                         ^ (SQL: select * from "mainvoice" where "merchantagreement_id" = ? order by "created_at" desc limit 1) (Bindings: array (   0 => 6, ))' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php:556 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select * from "...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Connection.php(277): Illuminate\Database\Connection->run('select * from "...', Array, Object(Closure)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1028): Illuminate\Database\Connection->select('select * from "...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1018): Illuminate\Database\Query\Builder->runSelect() #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php(1005): Illuminate\Database\Query\Builder->getFresh(Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(441): Illuminate\Database\Query\Builder->get(Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(132): Illuminate\Database\Eloquent\Builder->getModels(Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(108): Illuminate\Database\Eloquent\Builder->get(Array) #8 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(677): Illuminate\Database\Eloquent\Builder->first() #9 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(131): Controllers\Admin\MainvoiceController->getLastbalance(6) #10 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #11 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #15 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #17 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #18 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #19 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #20 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #21 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #22 {main} [] []
[2014-03-10 11:53:37] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ','' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:131 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 12:58:12] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 12:59:34] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 12:59:36] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 12:59:37] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 12:59:37] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 12:59:37] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 12:59:37] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:01:43] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:03:05] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:19] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:29] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:31] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:54] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:54] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:54] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:54] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:54] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:54] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:54] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:55] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:55] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:55] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:04:55] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:05:29] log.ERROR: exception 'ErrorException' with message 'Undefined variable: payable_sum' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:352 Stack trace: #0 /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2(352): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 352, Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/arp_de...') #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/arp_de...', Array) #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/arp_de...', Array) #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/arp_dev/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'getShow', Object(Illuminate\View\View)) #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getShow', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('1006') #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2014-03-10 13:05:56] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'endforeach' (T_ENDFOREACH)' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:347 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 13:13:05] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '$invoice' (T_VARIABLE)' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:237 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 13:13:32] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:242 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 14:50:54] log.ERROR: exception 'ErrorException' with message 'Undefined variable: merchant' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:652 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(652): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 652, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(216): Controllers\Admin\MainvoiceController->getLastbalance(6) #2 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #3 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #14 {main} [] []
[2014-03-10 14:51:00] log.ERROR: exception 'ErrorException' with message 'Undefined variable: merchant' in /var/www/arp_dev/app/controllers/admin/MainvoiceController.php:652 Stack trace: #0 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(652): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/arp_de...', 652, Array) #1 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(216): Controllers\Admin\MainvoiceController->getLastbalance(6) #2 /var/www/arp_dev/app/controllers/admin/MainvoiceController.php(65): Controllers\Admin\MainvoiceController->generate() #3 [internal function]: Controllers\Admin\MainvoiceController->getIndex() #4 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #5 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getIndex', Array) #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getIndex', Array) #7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #9 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #14 {main} [] []
[2014-03-10 16:02:52] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Laravel\Validator' not found' in /var/www/arp_dev/app/validator.php:9 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:03:43] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'lluminate\Support\Facades\Validator' not found' in /var/www/arp_dev/app/validator.php:9 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:04:30] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ''holdback_length'' (T_CONSTANT_ENCAPSED_STRING), expecting ')'' in /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php:39 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:10:41] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ''holdback_length'' (T_CONSTANT_ENCAPSED_STRING), expecting ')'' in /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php:39 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:11:13] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'holdback_length' (T_STRING), expecting ')'' in /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php:39 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:12:04] log.ERROR: exception 'BadMethodCallException' with message 'Method [validateRequiredIfAttribute] does not exist.' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php:1908 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(264): Illuminate\Validation\Validator->__call('validateRequire...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(264): Illuminate\Validation\Validator->validateRequiredIfAttribute('holdback_percen...', '2', Array, Object(Illuminate\Validation\Validator)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(225): Illuminate\Validation\Validator->validate('holdback_percen...', 'required_if_att...') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(239): Illuminate\Validation\Validator->passes() #4 /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php(131): Illuminate\Validation\Validator->fails() #5 [internal function]: Controllers\Admin\MerchantagreementController->postCreate() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postCreate', Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postCreate', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-03-10 16:13:09] log.ERROR: exception 'BadMethodCallException' with message 'Method [validateRequiredIfAttribute] does not exist.' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php:1908 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(264): Illuminate\Validation\Validator->__call('validateRequire...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(264): Illuminate\Validation\Validator->validateRequiredIfAttribute('holdback_percen...', '2', Array, Object(Illuminate\Validation\Validator)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(225): Illuminate\Validation\Validator->validate('holdback_percen...', 'required_if_att...') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(239): Illuminate\Validation\Validator->passes() #4 /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php(131): Illuminate\Validation\Validator->fails() #5 [internal function]: Controllers\Admin\MerchantagreementController->postCreate() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postCreate', Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postCreate', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-03-10 16:13:16] log.ERROR: exception 'BadMethodCallException' with message 'Method [validateRequiredIfAttribute] does not exist.' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php:1908 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(264): Illuminate\Validation\Validator->__call('validateRequire...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(264): Illuminate\Validation\Validator->validateRequiredIfAttribute('holdback_percen...', '2', Array, Object(Illuminate\Validation\Validator)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(225): Illuminate\Validation\Validator->validate('holdback_percen...', 'required_if_att...') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(239): Illuminate\Validation\Validator->passes() #4 /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php(131): Illuminate\Validation\Validator->fails() #5 [internal function]: Controllers\Admin\MerchantagreementController->postCreate() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postCreate', Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postCreate', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-03-10 16:14:45] log.ERROR: exception 'BadMethodCallException' with message 'Method [validateRequiredIfAttribute] does not exist.' in /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php:1908 Stack trace: #0 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(264): Illuminate\Validation\Validator->__call('validateRequire...', Array) #1 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(264): Illuminate\Validation\Validator->validateRequiredIfAttribute('holdback_percen...', '2', Array, Object(Illuminate\Validation\Validator)) #2 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(225): Illuminate\Validation\Validator->validate('holdback_percen...', 'required_if_att...') #3 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Validation/Validator.php(239): Illuminate\Validation\Validator->passes() #4 /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php(131): Illuminate\Validation\Validator->fails() #5 [internal function]: Controllers\Admin\MerchantagreementController->postCreate() #6 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array) #7 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('postCreate', Array) #8 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'postCreate', Array) #9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}() #10 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #11 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #12 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #13 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #14 /var/www/arp_dev/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #15 /var/www/arp_dev/public/index.php(49): Illuminate\Foundation\Application->run() #16 {main} [] []
[2014-03-10 16:18:59] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'if' (T_IF), expecting function (T_FUNCTION)' in /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php:42 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:19:34] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'if' (T_IF), expecting function (T_FUNCTION)' in /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php:42 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:20:31] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'if' (T_IF), expecting function (T_FUNCTION)' in /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php:42 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:20:43] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'if' (T_IF), expecting function (T_FUNCTION)' in /var/www/arp_dev/app/controllers/admin/MerchantagreementController.php:42 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:42:06] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:196 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:42:41] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:196 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
[2014-03-10 16:42:53] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in /var/www/arp_dev/app/storage/views/2d8971310e59d66b3c8ffc54892f57e2:196 Stack trace: #0 [internal function]: Illuminate\Exception\Handler->handleShutdown() #1 {main} [] []
